### DISCUSSION TITLE
Biglybt, a P2P app when opened in a sandbox... Opens outside of sandboxie.

### DISCUSSION THREAD

**Discussion by Matblanket** [sandboxie-plus/Sandboxie#1787] [Documents and notes]
When I tried to open Biglybt in sandboxie, the application instead opened outside of sandboxue

1. Install biglybt outside sandboxie
2. Try to launch it from a hardened sandbox
3. See error

**Expected behavior**
I expected it to open inside the sandbox like all other apps do.

**System details and installed software**
 -Windows 10 home 10.0.19043
 -windows defender (real time protection is enabled)

**Sandboxie configuration**
The sandbox was a default hardened sandbox.

**Comment by Dyras** [comment]
Are you using the template for this? It works flawlessly for me at least.
![grafik](https://user-images.githubusercontent.com/12563436/137689149-ea6ab2cd-0065-4b77-94ba-62db497390d2.png)

How exactly are you running it? I tried without a template now. I just right clicked biglybt.exe and chose run in Hardened and it still works.
![grafik](https://user-images.githubusercontent.com/12563436/137689688-b04c2e69-f6d0-4ccc-a6ec-5826f49621dc.png)

**Comment by DavidXanatos** [comment]
I have tested it and as far as I can tell everything works fine.
The opening outside sandbox happens only when you already have a un-sandboxed instance open and this as it looks to me is caused by the program using a localhost network socket to find already runnign instances.
This can not be blocked as when the port is blocked the tool does nto start at all.
So i would recommend to kill the un-sandboxed instance, and set it to be forced to always run sandboxed

**Comment by Matblanket** [comment]
> So i would recommend to kill the un-sandboxed instance, and set it to be forced to always run sandboxed

Yea, it seems like it was running unsandboxed and I didn't notice.
Anyways, thanks so much for all your work, I really appreciate it.
